src/test/java/org/dbunit/dataset/builder/DataSetBuilderIntegrationTest.java

```diff
@@ -62,6 +62,7 @@ public void testBuild() throws Exception {
 
         ReplacementDataSet expected = new ReplacementDataSet(new FlatXmlDataSetBuilder().build(
                 this.getClass().getResourceAsStream("/reference.xml")));
+        expected.addReplacementObject("[EPOCH_TIME]", new Date(0));
```

Comment:
I think to create a date where the time is set to 00:00:00 is a cleaner approach. Furthermore I'm unable to reproduce the issue. For me all test a green even in GMT+2.